src/containers/profile-page/components/mobile/ProfileHeader.module.css

```diff
@@ -49,7 +49,7 @@
   position: absolute;
   top: 21px;
   right: 13px;
-  box-shadow: 0 1px 4px 0 rgba(0,0,0,0.3);
+  box-shadow: 0 1px 4px 0 rgba(0, 0, 0, 0.3);
```

Comment:
Didn't touch this file in the end, but autoformatting took care of some problems. I figure might as well leave it in.